concord-server/server/src/main/java/com/rabidgremlin/concord/resources/VotesResource.java

```diff
@@ -0,0 +1,44 @@
+package com.rabidgremlin.concord.resources;
+
+import java.util.List;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.codahale.metrics.annotation.Timed;
+import com.rabidgremlin.concord.api.UserVotesMade;
+import com.rabidgremlin.concord.dao.VotesDao;
+
+@Path("votes")
```

Comment:
Would be more semantic if this was "users" so the full path suffix would be "users/scores"

Reply:
I was debating this too, but also makes sense for votes to have scores ???

Reply:
The vote scores are what we use to do the 'completed phrase' logic, whereby vote scores are user independent